stores/sql/database.go

```diff
@@ -146,4 +167,16 @@ type (
 		FCID        FileContractID
 		RenewedFrom FileContractID
 	}
+
+	ContractInfo struct {
```

Comment:
I think this type is only used in one place (`UpdateContract`). If so, I'd inline it. Goal is to keep dedicated types to a minimum